Formula/libetonyek.rb

```diff
@@ -45,7 +45,7 @@ def install
                           "--disable-werror",
                           "--disable-tests",
                           "--prefix=#{prefix}",
-                          "--with-mdds=1.5"
+                          "--with-mdds=#{Formula["mdds"].version.major_minor}"
```

Comment:
Looking at this again, I'm not sure if this is necessarily correct.

This needs the "API version", but mdds 1.7.0 shipped with API version 1.5.

The only way to do it manually would be to do some file/directory scanning.

Reply:
I see - the `2.0` in the `mdds-2.0.pc` is the API version, not the version of `mdds`.  It happens to be same right now, but I imagine we're anticipating that `mdds` 2.1 may still have API version 2.0 and it would still use `mdds-2.0.pc`, not `mdds-2.1.pc`.

I think the right way to do this automatically is to extract the `2.0` from `mdds-2.0.pc`.  That will take a bit more code than I've used here but seems feasible.

Reply:
I fixed this up so that it now automatically extracts the API version of the .pc file.